interviews/backend-takehome.md

```diff
@@ -0,0 +1,43 @@
+# Backend Takehome Grading
+
+## Section Scores
+
+### Solution Completion
+
+All requirements are met as outlined by the assignment's requirements.
+
+- 💖 4 - Perfect
+- 👍 3 - One minor bug with flow using expected inputs, or a couple bugs with validation of corner cases
+- 👎 2 - Two or more missed requirements
+- 🛑 1 - Several requirements missed; compile errors that are not trivial to resolve
+
+### Code Quality
+
+Code is generally well-organized and easy to understand.
+
+- 💖 4 - Well-organized, DRY, easily readable, good variable/function names, would pass linter w/minimal issues
+- 👍 3 - Minor repetition in logic, a couple confusing variable/function names, minor formatting issues
+- 👎 2 - Not idiomatic for language chosen, difficult to understand and trace
+- 🛑 1 - Unable to understand, non-standard formatting, variable names difficult to follow
+
+### Data structures and runtime
+
+Data structures are chosen appropriately for low complexity and low memory allocation
+
+- 💖 4 - Appropriate data structures; efficient use of conditionals/loops; minimal array scanning; appropriate passing of parameters and return values
+- 👍 3 - Reasonable use of data structures; some opportunity for reduction of costly repeated operations; some inefficiency in usage of parameters and return values
+- 👎 2 - Excessive hardcoding of control flow and repetition of the same logic
+- 🛑 1 - Can enter unintended infinite loops; may not function properly; shows lack of understanding of data structures
+
+### Testing
+
+While unit tests are not required, some form of test that verifies that the code is working appropriately needs to exist
```

Comment:
This doesn't correlate with wording in the backend prompt: [Unit tests are written to determine that code is correct.](https://github.com/Codecademy/internship-code-challenge/tree/master/backend#how-you-will-be-evaluated)

And Michael's [comment here](https://codecademy.slack.com/archives/GQWH1EPJ7/p1580152302013600) leads me to believe we haven't led to a consensus yet.

I'm for `unit tests or script`, but we can have that discussion in the appropriate channel.

Reply:
Aha, excellent callout! I'll adjust here...